index.js

```diff
@@ -136,21 +137,31 @@ class TagInput<T> extends React.PureComponent<Props<T>, State> {
     regex: DEFAULT_TAG_REGEX,
     tagColor: '#dddddd',
     tagTextColor: '#777777',
+    inputDefaultWidth: 90,
     inputColor: '#777777',
     maxHeight: 75,
     parseOnBlur: false,
   };
 
-  static inputWidth(text: string, spaceLeft: number, wrapperWidth: number) {
+  static inputWidth(text: string, spaceLeft: number, inputDefaultWidth: number, wrapperWidth: number) {
```

Comment:
Please break this down into multiple lines as follows in order to keep all lines less than 80 characters:
```js
  static inputWidth(
    text: string,
    spaceLeft: number,
    inputDefaultWidth: number,
    wrapperWidth: number,
  ) {
```